### ISSUE TITLE
section label: set colour using border

### ISSUE THREAD

**Issue by andrew-nowak** [BBC-archive/psammead#605]
outcome of #558 
**Is your feature request related to a problem? Please describe.**
In Windows High Contrast mode, the line in the section label disappears. Instead of using a background colour, we should set the colour of this element using a `border`.

**Describe the solution you'd like**
`` `border-top: ${1/16}rem solid ${C_PEBBLE};` ``
remove `height` and `background`.
Only set border-top (or border-bottom) to avoid doubling the height of the line

Users of HCM may be using a grey as their background colour, so in HCM we should use a safer colour; use the following example for inspiration

```css
@media screen and (-ms-high-contrast: active) {
  border-color: windowText;
}
```

**Describe alternatives you've considered**

**Testing notes**
The following accessibility testing should be done: points 4, 5, 6, & 7 under 'Checklist manual' on https://bbc-news.github.io/accessibility-news-and-you/accessibility-news-and-testers

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

**Additional context**
